server/reportmanager/management/commands/import_reports_from_bigquery.py

```diff
@@ -38,7 +38,9 @@ def handle(self, *args, **options):
                 FROM `{settings.BIGQUERY_TABLE}` as r
                 LEFT JOIN `{settings.BIGQUERY_TRANSLATIONS_TABLE}` t
                     ON r.uuid = t.report_uuid
-                WHERE r.reported_at >= @since;""",
+                WHERE r.url IS NOT NULL
```

Comment:
I think it would be nice to have at least a comment about why we're excluding some reports here. I'm not too worried because null URL or comment suggests a broken report, but arguably the ETL isn't the right layer to do that validation.